### ISSUE TITLE
line 1087

### ISSUE THREAD

**Issue by LastDance500** [zhiwehu/Python-programming-exercises#114]
the if in the loop maybe wrong?
tp=(1,2,3,4,5,6,7,8,9,10)
li=list()
for i in tp:    
	if i%2==0:      #<-----tp[i] should be i  
		li.append(i)

tp2=tuple(li)
print (tp2)

**Comment by n16i9kita** [comment]
yes, I think there is an error

**Comment by nish-ant** [comment]
Yes, either that or `i` can iterate over the indices of `tp`, keeping rest of the code unchanged : `for i in range(len(tp))`

**Comment by chirag-list** [comment]
tp2=(2,4,6,8,10)

**Comment by bonnykato** [comment]
Is not wrong but indentation of code after loop  is wrong, it supposed to be  inside of loop

Check out;

tp=(1,2,3,4,5,6,7,8,9,10)
li=list()
for i in tp:
if i%2==0: 
     li.append(i)  # indentation
